src/components/Main/Containment/ContainmentGraph.tsx

```diff
@@ -162,57 +160,71 @@ function draw({ data, width, height, onDataChange, d3ref }: DrawParams) {
     .attr('cy', d => yScale(d.y))
     .attr('r', 8)
 
-  const tooltip = graph.append('g');
-
-  const callout = (g, value) => {
-      if (!value) return g.style("display", "none");
-
-      g.style("display", null)
-       .style("pointer-events", "none")
-       .style("font", "12px sans-serif");
-
-      const text = g.selectAll("text")
-        .data([null])
-        .join("text")
-        .call(text => text
-          .selectAll("tspan")
-          .data((value + "").split(/\n/))
-          .join("tspan")
-            .attr("x", 0)
-            .attr("y", (d, i) => `${i * 1.1}em`)
-            .style("font-weight", (_, i) => i ? null : "bold")
-            .text(d => d));
-
-      const {x, y, width: w, height: h} = text.node().getBBox();
+  const tooltip = graph.append('g')
+
+  const callout = (g: d3.Selection<SVGGElement, unknown, null, undefined>, value: string | null) => {
+    if (!value) return g.style('display', 'none')
+
+    g.style('display', null)
+      .style('pointer-events', 'none')
+      .style('font', '12px sans-serif')
+
+    const text = g
+      .selectAll<SVGTextElement, TimePoint>('text')
+      .data([null])
+      .join('text')
+      .call(text =>
+        text
+          .selectAll('tspan')
+          .data(value.split(/\n/))
+          .join('tspan')
+          .attr('x', 0)
+          .attr('y', (d, i) => `${i * 1.1}em`)
+          .style('font-weight', (_, i) => (i ? null : 'bold'))
+          .text(d => d),
+      )
+
+    const node = text.node()
```

Comment:
Here I have to handle the situation where `text.node()` returns `null`. I decided to throw an error as this would certainly be a bug. Previously, `text.node()` was assumed to always return an object.